src/functions/module_custom.ts

```diff
@@ -43,3 +43,23 @@ export const FORMAT_LARGE_NUMBER: AddFunctionDescription = {
     return toNumber(value);
   },
 };
+
+// -----------------------------------------------------------------------------
+// HYPERLINK
+// -----------------------------------------------------------------------------
+export const HYPERLINK: AddFunctionDescription = {
+  description: _lt("Creates a hyperlink in a cell."),
+  args: args(`
+    url (string) ${_lt("The full URL of the link enclosed in quotation marks.")}
+    link_label (string, optional) ${_lt(
+      "The text to display in the cell, enclosed in quotation marks."
+    )}
+  `),
+  returns: ["STRING"],
+  compute: function (url: PrimitiveArgValue, linkLabel: PrimitiveArgValue): string {
+    const processedUrl = toString(url);
+    const processedLabel = toString(linkLabel) || toString(url);
+    return toString(`[${processedLabel}](${processedUrl})`);
+  },
+  isExported: true,
```

Comment:
add the function to the exported function xlsx test here https://github.com/odoo/o-spreadsheet/blob/16.0/tests/xlsx_export.test.ts#L115